lib/shoryuken/environment_loader.rb

```diff
@@ -92,6 +92,7 @@ def load_rails
         require File.expand_path('config/environment.rb')
       end
 
+      Shoryuken.options.merge!(config_file_options)
```

Comment:
@phstc I've updated the PR to fix this issue as well. The options passed from the CLI were destructively overwriting the options from the configuration file on merge, because an empty array was being passed.